Update Installing_OpenSearch_database.md (#3669)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

     reserved: true
     backend_roles:
     - "admin"
     description: "admin user"
   ```

##### Restart OpenSearch and apply the configuration

1. Restart OpenSearch:

   ```bash
   sudo systemctl restart opensearch
   ```

> 1. The settings in files */etc/opensearch/opensearch-security/\*.yml* are not applied to the cluster configuration immediately. The configuration is applied by running the script *securityadmin.sh*, which uses the super admin certificates generated earlier

Reviewer: ```suggestion
1. To apply the settings in the files */etc/opensearch/opensearch-security/\*.yml* to the cluster configuration, run the script *securityadmin.sh*, which uses the super admin certificates generated earlier:
```

Author: no objections from my side
